### ISSUE TITLE
Stale git information links

### ISSUE THREAD

**Issue by pnorman** [gravitystorm/openstreetmap-carto#2341]
From https://github.com/gravitystorm/openstreetmap-carto/pull/2334#issuecomment-245776407

> I would **much** rather point at external documentation. We are currently doing that in [workflow](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md#workflow) with a link to [github's pull request guidance](https://help.github.com/articles/about-pull-requests/).

> When we added the link it contained text about the fork & pull model but it's not there anymore. We could link to https://help.github.com/categories/collaborating-on-projects-using-issues-and-pull-requests/. The fork and pull model is defined in https://help.github.com/articles/about-collaborative-development-models/

> The two relevant parts of that category seem to be https://help.github.com/articles/working-with-forks/ and https://help.github.com/articles/proposing-changes-to-a-project-with-pull-requests/. Would linking to those two work?

**Comment by Ircama** [comment]
Yes, and other than linking general documentation, I would also suggest actualizing it to this repo for improved guidance.

**Comment by kocio-pl** [comment]
This page is also not available now:

https://help.github.com/articles/proposing-changes-to-a-project-with-pull-requests/

instead we could probably link to this one:

https://help.github.com/articles/creating-a-pull-request-from-a-fork/

**Comment by kocio-pl** [comment]
Do we still want to add some more links there? 

Current link in the Workflow section is working, though it's a redirect to the URL ending with a "/" and I like to avoid redirecting whenever possible, since this indicates that URL is not "canonical" now and might be silently dropped one day.

**Comment by dch0ph** [comment]
@imagico I think this issue has evaporated with time.

**Comment by imagico** [comment]
Yes, links in CONTRIBUTING.md seem to be all working now.
